apps/web/src/main.tsx

```diff
@@ -8,12 +8,12 @@ import { BrowserRouter } from "react-router-dom";
 import { PersistGate } from "redux-persist/integration/react";
 
 import { App } from "./components/App";
+import { ErrorPage } from "./components/ErrorPage/ErrorPage";
```

Comment:
we can import directly from .`/components/ErrorPage` since we export ErrorPage.tsx from ErrorPage/index.js

Reply:
We should update lint rules with something like this

```{
  "plugins": ["import"],
  "rules": {
    "import/no-useless-path-segments": ["error", { "commonjs": true }]
  }
}
```